test/connection-manager/auto-dial.spec.ts

```diff
@@ -51,6 +52,7 @@ describe('auto-dial', () => {
 
     const connectionManager = stubInterface<ConnectionManager>()
     connectionManager.getConnectionsMap.returns(new PeerMap())
+    connectionManager.getDialQueue.returns([])
```

Comment:
the [getDialQueue mock](https://github.com/libp2p/js-libp2p-interfaces/blob/master/packages/interface-mocks/src/connection-manager.ts#LL192C1-L193C1) should be sufficient here.